### PR TITLE
Updated components/aws/sagemaker/README.md 

### PR DESCRIPTION
**Pull request by IvyBazan**

### CODE REVIEW

**Comment by Jeffwan** [comment]
@IvyBazan Can you help rebase the changes

**Comment by IvyBazan** [comment]
@Jeffwan Fixed merge conflict

File: components/aws/sagemaker/README.md [hunk 1]
```diff
@@ -3,10 +3,11 @@
 ## Summary
 With Amazon SageMaker Components for Kubeflow Pipelines (KFP), you can create and monitor training, tuning, endpoint deployment, and batch transform jobs in Amazon SageMaker. By running Kubeflow Pipeline jobs on Amazon SageMaker, you move data processing and training jobs from the Kubernetes cluster to Amazon SageMaker’s machine learning-optimized managed service. The job parameters, status, logs, and outputs from Amazon SageMaker are still accessible from the Kubeflow Pipelines UI.
 
-
 ## Components
 Amazon SageMaker Components for Kubeflow Pipelines offer an alternative to launching compute-intensive jobs in Kubernetes and integrate the orchestration benefits of Kubeflow Pipelines. The following Amazon SageMaker components have been created to integrate 6 key Amazon SageMaker features into your ML workflows. You can create a Kubeflow Pipeline built entirely using these components, or integrate individual components into your workflow as needed. 
 
+For an end-to-end tutorial using these components, see [Using Amazon SageMaker Components](https://sagemaker.readthedocs.io/en/stable/workflows/kubernetes/using_amazon_sagemaker_components.html).
```
Comment (surajkota) [inline components/aws/sagemaker/README.md:9]: thanks, this is a good change, lets add reference to https://github.com/kubeflow/pipelines/tree/master/samples/contrib/aws-samples as well ?

Comment (IvyBazan) [reply components/aws/sagemaker/README.md:9]: Done

**Comment by IvyBazan** [comment]
@surajkota This PR is still waiting on approval to test

**Pull request merged**